crates/gen/src/class.rs

```diff
@@ -236,7 +240,7 @@ impl Class {
                 }
                 impl ::std::convert::From<&#from> for #into {
```

Comment:
I'm not sure how useful `TryFrom` would be since these conversions can never fail - they are required by metadata - so I wouldn't want to give the developer the impression that this is something they should check for.

Reply:
Ok that makes sense. If we're sure this can't ever fail (lest it be a bug on our part), maybe we should consider using [`unreachable`](https://doc.rust-lang.org/std/macro.unreachable.html) or even the unsafe `unreachable_unchecked`.

Reply:
Created an issue for this: https://github.com/microsoft/winrt-rs/issues/356